apps/site/package.json

```
@@ -119,5 +119,8 @@
       "./*.mjs",
       "./*/index.mjs"
     ]
+  },
+  "engines": {
+    "node": ">=24"
```

```suggestion
    "node": "24.x"
```
I think we should make this `24.x` so that we're not inadvertently allowing `25.x` (or `26.x`) in the future? Using `24.x` is what Vercel recommends: https://vercel.com/docs/functions/runtimes/node-js/node-js-versions#version-overrides-in-package.json

---

Noted. Ideally the Vercel default should _also_ be set post this PR

---

Yah, agree, once this is in we can bump the default, this just ensures this PR builds correctly and I think its a good habit to have encoded for future version bumps too.